dll/win32/browseui/lang/zh-CN.rc

```diff
@@ -315,7 +315,7 @@ END
 
 STRINGTABLE
 BEGIN
-    IDS_BROWSEFORNEWTOOLAR "Choose a folder"
-    IDS_TOOLBAR_ERR_TITLE "Toolbar"
-    IDS_TOOLBAR_ERR_TEXT "Cannot create toolbar."
+    IDS_BROWSEFORNEWTOOLAR "选择一个文件夹"
+    IDS_TOOLBAR_ERR_TITLE "工具栏"
+    IDS_TOOLBAR_ERR_TEXT "无法创建新的工具栏"
```

Comment:
Shouldn't that last one finish with a dot?

Reply:
Not all languages in the world have a concept of "full stop" in their sentences.

Reply:
I've seen chinese and japanese sentences finishing with a "round full stop" : "。" (dunno the exact name) in the translations elsewhere, hence my question :)

Reply:
I see, yes, apparently Chinese and Japanese both have the "ideographic full stop" symbol used sometimes. But I think in both languages it just exists for convenience, to split sentences in a large paragraph, I don't believe it's necessary to use it in short lines like this. but since I could be wrong, I'll let the author confirm.